.github/CONTRIBUTING.md

```diff
@@ -77,6 +77,23 @@ ok      github.com/containous/traefik   0.005s  coverage: 4.1% of statements
 Test success
 ```
 
+For development purpose, you can specifiy which tests to run by using:
+```
+# Run every tests in the ConsulCatalog suite
+TESTFLAGS="-check.f MyTestSuite" make test-integration
```

Comment:
Hum small nit, but the comment is a little bit wrong :stuck_out_tongue_closed_eyes: (`MySuite` vs `ConsulCatalog` suite)